uwblueprint/community-fridge-kw
# fix: verify email spinner

## Implementation description. How did you make this change?
* Before, when user tried to verify email, the spinner kept on showing. Now if an error occurs, it shows an error message otherwise it should work as expected



## Steps to test
1. Before testing, make sure to go on firebase and change the action url to local: http://localhost:3000/action
2. Create an account
3. Verify email, make sure you dont just see the spinner but can see the confirmation email screen
4. Try using the verify link again and make sure u see the error msg
5. Try forget pw flow as well, cuz there were some changes made in Action.tsx file to make sure i didnt break that flow




## Checklist
